docs/setup-cluster/on-prem/requirements.rst

```diff
@@ -140,3 +141,12 @@ Install on macOS
 
 Docker on macOS does not support containers that use GPUs. Because of this, macOS Determined agents
 are only able to run CPU-based workloads.
+
+.. _install-docker-on-WSL:
+
+Install on Windows Subsystem for Linux (WSL)
+============================================
+
+Follow the steps above for Linux to install Docker and the NVIDIA Container Toolkit on your Linux 
+distribution in your WSL distribution. 
+
```

Comment:
.. _install-docker-on-linux:

Install on Linux
============

...


.. _install-docker-on-WSL:

Install on Windows Subsystem for Linux (WSL)
============================================

Follow the steps for installing Docker on Linux. :ref:`installing Docker on Linux <install-docker-on-linux>`.